django/contrib/gis/db/models/fields.py

```diff
@@ -368,3 +369,40 @@ class ExtentField(GeoSelectFormatMixin, Field):
 
     def get_internal_type(self):
         return "ExtentField"
+
+
+class RasterField(Field):
+    """
+    Raster field for GeoDjango
+    """
+
+    description = "Raster Field"
+    geom_type = 'RASTER'
```

Comment:
This is not ideal naming, but it leverages the `geom_type` attribute checks in  the operations module, so renaming this to be more general (to include raster) would require updating the name in the geometry related code as well.